.circleci/config.yml

```diff
@@ -193,7 +193,7 @@ jobs:
       - checkout
       - run: brew install swiftlint swiftformat
       - run: swiftlint --strict
-      - run: swiftformat --lint --swiftversion 4 --verbose megazords
+      - run: swiftformat megazords components/*/ios --lint --swiftversion 4 --verbose
```

Comment:
Putting more than one argument at the end here seems to hit a bug in its arg parsing that says '--verbose cannot take an argument' or something. Putting them at the start works.